stout_a05a.js

```diff
@@ -0,0 +1,29 @@
+/*Kyle Stout
+  stout_a05a
+  INFO 2124WW
+  Thoendel
+  1-19-2020
+*/
+
+let todolist = ["Eat", "Sleep", "Shower", "Brush Teeth"];
+todolist;

```

Comment:
to output this to the console - you'd want to use console.log(todolist);
-1

Reply:
![image](https://user-images.githubusercontent.com/54589936/72817055-c33df080-3c2e-11ea-95f0-d3ecfa973e7f.png)
As is - your script doesn't output anything.

Reply:
How come in node if i just put todolist; it prints it? Im guessing thats specific to node and not just js in general?

Reply:
Actually - running  your script with node - it doesn't output anything - as in my screenshot above. You must be referring to the interactive cli - which is going to perform differently for debugging purposes. Does that help?

Reply:
Yes it does. I was thinking in terms of the node cli and not the script itself which had to be told to print something.